### PR TITLE
add cloud-provider-aws admin/maintainers teams for k8s.io/cloud-provider-aws

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kim.andrewsy@gmail.com>

cc @justinsb @nckturner

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -1281,6 +1281,20 @@ teams:
     - lavalamp
     - sttts
     privacy: closed
+  cloud-provider-aws-admins:
```
Comment (nikhita) [inline config/kubernetes/org.yaml:1284]: Can this be moved to https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-cloud-provider/teams.yaml? That way you can approve PRs touching this team directly :)

Comment (justaugustus) [reply config/kubernetes/org.yaml:1284]: +1

Comment (andrewsykim) [reply config/kubernetes/org.yaml:1284]: Yes, good call let me update

Comment (andrewsykim) [reply config/kubernetes/org.yaml]: updated

**Comment by nikhita** [comment]
/lgtm

/hold
in case you want more eyes

**Comment by justaugustus** [comment]
/lgtm
/approve

@kubernetes/owners -- When you have a chance to, please lift the hold and enact the repo changes.

**Comment by nikhita** [comment]
Teams have been granted access

**Pull request merged**